webapp-django/package.json

```diff
@@ -10,9 +10,13 @@
   "license": "MPL-2.0",
   "devDependencies": {
     "cssmin": "0.4.3",
+    "eslint": "^4.19.1",
```

Comment:
We prefer to pin our dependencies to a specific version. We should remove the [caret range operator `^`](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004) which allows minor and patch version updates without our explicit approval.